### PR TITLE
fix(table): fix a11y

### LINKED ISSUE #1564: Fix a11y in Table
3 errors found in: /components/Table/examples-full/
1) ARIA attributes must conform to valid values -- https://dequeuniversity.com/rules/axe/3.1/aria-valid-attr-value -- impact: critical
2) IDs used in ARIA and labels must be unique -- https://dequeuniversity.com/rules/axe/3.1/duplicate-id-aria -- impact: critical
3) Form elements must have labels -- https://dequeuniversity.com/rules/axe/3.1/label -- impact: critical

### PR DESCRIPTION
**Pull request by christiemolloy**
fix #1564

### CODE REVIEW

File: src/patternfly/components/Table/examples/table-width-example.hbs [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <input type="checkbox" name="checkrow2" aria-labelledby="width-node2">
       {{/table-td}}
       {{#> table-th table-th--data-label="Repository Name"}}
-        <p id="{{table--id}}-node2">Node 2</p>
+        <p id="width-node2">Node 2</p>
```
Comment (srambach) [inline src/patternfly/components/Table/examples/table-width-example.hbs]: Why not do the `table--id` as the other examples have?

Comment (christiemolloy) [reply src/patternfly/components/Table/examples/table-width-example.hbs]: good point will update it while im at it

**Comment by srambach** [comment]
Looks good to me.

**Comment by christiemolloy** [comment]
@mcoker this looks awesome - thanks!

**Pull request merged**